apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRowHeader.tsx

```diff
@@ -36,7 +36,7 @@ const PolicyTableRowHeader = ({
   return (
     <div id={table.id.toString()} className="flex w-full items-center justify-between">
       <div className="flex gap-x-4 text-left">
-        <Link href={`/project/${ref}/editor/${table.id}`} className="flex items-center gap-x-2">
+        <Link href={`/project/${ref}/editor/${table.id}?schema=${table.schema}`}> className="flex items-center gap-x-2">
```

Comment:
<img width="474" alt="Screenshot 2024-10-14 at 10 18 52" src="https://github.com/user-attachments/assets/c6e5a187-ec45-429b-9f07-b24a7c63d96e">

```suggestion
        <Link href={`/project/${ref}/editor/${table.id}?schema=${table.schema}`} className="flex items-center gap-x-2">
```

Reply:
Fixed. Can you take another look, @alaister?